build.gradle

```diff
@@ -50,18 +47,14 @@ gitProperties {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter'
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
-	implementation 'org.springframework.cloud:spring-cloud-starter-openfeign'
 
 	implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.1.0'
-	implementation 'io.github.openfeign:feign-okhttp'
 	implementation 'org.codehaus.janino:janino'
-	implementation 'org.apache.commons:commons-lang3'
-	implementation 'commons-fileupload:commons-fileupload:1.5'
 	implementation 'org.yaml:snakeyaml:2.0'
 
-	testImplementation 'org.springframework.cloud:spring-cloud-stream-test-support'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation 'org.junit.jupiter:junit-jupiter-api'
 	testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
```

Comment:
Are these dependencies required?

Reply:
check also for mockito and junit dependency provided below.

Define a version for lombok (you are defining it twice)